strawberry/annotation.py

```diff
@@ -249,7 +267,7 @@ def _is_optional(cls, annotation: Any) -> bool:
     def _is_list(cls, annotation: Any) -> bool:
         """Returns True if annotation is a List"""
 
-        annotation_origin = getattr(annotation, "__origin__", None)
+        annotation_origin = get_origin(annotation)
```

Comment:
Also @patrick91 was there a reason such as performance to get these attributes from the annotations directly instead of using `get_origin`, `get_args` ? If so I can revert these changes quickly

Reply:
no, I don't think there was a particular reason 😊

Reply:
I think it is because `get_origin` and `get_args` were introduced on python 3.8, and probably weren't available on `typing_extensions` when this code was written.